### PR TITLE
Edited as discussed with @nomadscientist

### PR DESCRIPTION
**Pull request by MarisaJL**
I also need to add the extra pictures - I might have to do this separately

### CODE REVIEW

**Comment by MarisaJL** [comment]
Sorry - I was trying to edit the other pull request #3907 but this ended up as a separate one!

**Comment by shiltemann** [comment]
@MarisaJL to add to the existing PR, all you need to do is push to @nomadscientist's branch and it will be added automatically (or make the PR to her fork if you don't have permissions to push directly). 

So you could either close this and add to the other branch, or we wait until the other PR is merged and then merge this one, up to you (two) :)

**Review by nomadscientist — APPROVED** [review]
Looks ace!

**Pull request merged**
